### PR TITLE
Dynamic channels 4

### PR DESCRIPTION
**Pull request by norkans7**
based on #1418

### CODE REVIEW

File: temba/channels/types/dartmedia/type.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+class DartMediaType(ChannelType):
+    """
+    An DartMedia channel (http://dartmedia.biz/)
+    """
+
+    code = 'DA'
+    category = ChannelType.Category.PHONE
+
+    name = "DartMedia"
+    slug = "dart_media"
```
Comment (rowanseymour) [inline temba/channels/types/dartmedia/type.py]: Any reason to make this different to the package name "dartmedia"?

File: templates/channels/channel_claim.haml [hunk 2]
```diff
@@ -167,27 +167,6 @@
                 -blocktrans
                   Easily add a two way number you have configured with Hub9 in Indonesia.
 
-      .claim-row.dartmedia-channel{class:"{% if recommended_channel == 'dartmedia' %}recommended{% endif %}"}
```
Comment (rowanseymour) [inline templates/channels/channel_claim.haml]: Do we need to add a mechanism for channel types to be marked as recommended for users that meet a certain criteria?

Comment (norkans7) [reply templates/channels/channel_claim.haml]: what about adding the recommendation_blurb attribute on the ChannelType?

Comment (rowanseymour) [reply templates/channels/channel_claim.haml]: Yeah tho I'm looking at `Org.get_recommended_channel()` and it doesn't seem possible for dartmedia to ever be the recommended channel.

**Pull request merged**